clang-tools-extra/test/clang-tidy/checkers/readability/use-std-min-max.cpp

```diff
@@ -0,0 +1,230 @@
+// RUN: %check_clang_tidy %s readability-use-std-min-max %t
```

Comment:
Add `-- -- -fno-delayed-template-parsing` this is why windows tests is falling.
Additionally explicitly set c++ standard with `-std=c++11-or-later` like in other checks